### PR TITLE
Update document

### PR DESCRIPTION
**Pull request by janus**
Update file, RUNNING_LOCALLY_DOCKER.dm

### CODE REVIEW

**Comment by janus** [comment]
@owocki

**Comment by janus** [comment]
@owocki 
Please review.

File: docs/RUNNING_LOCALLY_DOCKER.md [hunk 1]
```diff
@@ -8,17 +8,35 @@ cp app/app/local.env app/app/.env
...
+The above would create a background daemon when it finished installation. It takes pretty long time, an hour or more. The good news is that it runs on its own. If you have error while running this it is likely to come from docker and docker-compose.
+
 ### Running in the foreground
 
 ```shell
 docker-compose up --build
 ```
+### Screens during building
+
+![alt text](https://raw.githubusercontent.com/janus/web/janus_doc/docs/imgs/screenshot1.png "Installation screenshot")
```
Comment (danlipert) [inline docs/RUNNING_LOCALLY_DOCKER.md]: Can you please include these files in this repo instead of linking it from your own? thanks!

Comment (janus) [reply docs/RUNNING_LOCALLY_DOCKER.md]: Thanks. I have moved the files to repo.

**Pull request merged**